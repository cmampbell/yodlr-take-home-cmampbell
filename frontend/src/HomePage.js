import React from 'react';

const HomePage = () => {
    return (
        <div>
            <h1>Welcome to Yodlr!</h1>
            <p>Visit the sign up page to register</p>
            <p>Visit the admin page to see all users</p>
        </div>
    );
}

export default HomePage;