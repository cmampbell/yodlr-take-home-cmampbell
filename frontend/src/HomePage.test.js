import React from "react";
import renderWithRouter from "./renderWithRouter";
import HomePage from "./HomePage";


describe('HomePage tests', () => {
    it('should render without crashing', () => {
        // define a full route as the first argument
        renderWithRouter(<HomePage />)
    })

    it('should match the snapshot', ()=> {
        const { asFragment } = renderWithRouter(<HomePage />)

        expect(asFragment()).toMatchSnapshot();
    })
})