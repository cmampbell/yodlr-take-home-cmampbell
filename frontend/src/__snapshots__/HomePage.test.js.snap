// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage tests should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Welcome to Yodlr!
    </h1>
    <p>
      Visit the sign up page to register
    </p>
    <p>
      Visit the admin page to see all users
    </p>
  </div>
</DocumentFragment>
`;
